__tests__/e2e/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "e2e-synthetics",
+  "private": true,
+  "main": "index.js",
+  "license": "MIT",
+  "scripts": {
+    "setup": "bash scripts/setup.sh",
+    "test": "npm i && npm run setup &&  bash scripts/test.sh"
+  },
+  "devDependencies": {
+    "@elastic/synthetics": "../../",
```

Comment:
seems wrong, needs the file protocol.

Reply:
we tried file protocol, i think it didn't work.

Reply:
Interesting, I just tested it and it works for me. file:../../ linking to the repo's `package.json` directory. Can you try once again?